src/main/ts/configs/Standard.ts

```diff
@@ -33,6 +33,13 @@ export const base: Linter.Config = {
       singleline: { delimiter: 'semi', requireLast: false }
     }],
     '@typescript-eslint/member-ordering': 'error',
+    '@typescript-eslint/naming-convention': [
```

Comment:
This seems like we really need more configuration for this, so I'd like us to investigate before we release this. For now, this is fine though so maybe we should just log a follow up task for this and to test how it works/what issues it flags in say TinyMCE?

Reply:
I added this to the tinymce repo and it only detects one type inside alloy that needs to change https://github.com/tinymce/tinymce/blob/develop/modules/alloy/src/main/ts/ephox/alloy/api/system/Gui.ts#L37.
But not sure about other repos outside tinymce however since we use lock files everywhere we could fix them as we go or just disable the rule if the whole repo uses lowercase types for some odd reason.

Reply:
We'll follow up with other naming in a separate PR (or maybe even a later version).